hat/assets/js/apps/Iaso/domains/orgUnits/components/OrgUnitsFilters.tsx

```diff
@@ -236,14 +234,13 @@ export const OrgUnitFilters: FunctionComponent<Props> = ({
                         }
                     />
                 </Box>
-                <SearchFilter
-                    withMarginTop
-                    uid={`search-${searchIndex}`}
-                    onEnterPressed={() => onSearch()}
-                    onChange={handleChange}
+                <InputComponent
                     keyValue="search"
-                    value={filters?.search ? `${filters?.search}` : ''}
-                    onErrorChange={setTextSearchError}
```

Comment:
The original component has an error state. It is absent from your refactor.
This will cause a regression, since there's custom logic in the original component so it is put in error if the user inputs certain characters. Removing this logic will cause a regression and reintroduce a bug